shared/src/main/scala/coop/rchain/pcasper/finalization/Finalizer.scala

```diff
@@ -0,0 +1,6 @@
+package coop.rchain.pcasper.finalization
+
+trait Finalizer[F[_], M, S, FT <: Final[M, S]] {
```

Comment:
If these short names of generic parameters are used, maybe better to add a description in the comment above? M - Message, S - Sender, FT - Final trait.